backend/core/common/validators.py

```
@@ -0,0 +1,46 @@
+	try:
+		result = email_validate(normalized, check_deliverability=False)
+		return result.normalized
+	except EmailNotValidError as exc:
+		raise ValidationError("Invalid email format") from exc
+
+
+def validate_password(password: str) -> str:
+	"""Validate password constraints used for registration."""
+	clean = sanitize_text(password)
+	if len(clean) < MIN_PASSWORD_LEN or len(clean) > MAX_PASSWORD_LEN:
+		raise ValidationError("Password must be between 8 and 128 characters")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)